components/coachindicator/stories/coachindicator.stories.js

```diff
@@ -25,7 +28,7 @@ export default {
 				type: { summary: "string" },
 				category: "Component",
 			},
-			options: ["dark", "light"],
+			options: ["default", "dark", "light"],
```

Comment:
Be sure to add the default value to the args as well on line 35: `variant: "default"`

Reply:
oh yes, good catch!